drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -445,6 +458,21 @@ def _validate_do_attempt(self):
                 self.i18n_service.gettext("Max number of attempts reached")
             )
 
+    def _validate_show_answer(self):
```

Comment:
@arbrandes Is this called from multiple places? If not, I think it would be perfectly fine to move the checks to the `show_answer` handler and remove this method.